notebooks/214-grammar-correction/214-grammar-correction.ipynb

```diff
@@ -0,0 +1,500 @@
+{
```

Comment:
<p>This textarea doesn't look good. Is there any possibility to make it larger by default?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
![image](https://user-images.githubusercontent.com/4547501/220148648-c403055b-5a06-4030-a03a-98821c05ea32.png)

Reply:
addaed layout auto for expanding on whore screen